test/browser/component.js

```diff
@@ -499,22 +499,30 @@ describe(`Nested Component instance with partially shared state`, function() {
 
 describe(`Rendering exception`, function() {
   let el;
+  let eventSpy;
 
   beforeEach(async function() {
+    WCT._config.trackConsoleError = false;
```

Comment:
this is the ugly workaround for console errors (see: https://github.com/Polymer/tools/issues/3080). if there's a better way to do this, lmk

Reply:
If the only point is to stub/spy on console.error then retaining the existing generic `logError()` method implementation is fine. IIRC your previous version converted it into a render-error-specific logger. So the new code would look something like:
```js
logError() {
  console.error(...arguments);
}

onRenderError(error) {
  const message = `Error while rendering ${this.toString()}`;
  this.logError(message, this, `\n`, error);
  this.el.dispatchEvent(new CustomEvent(`rendererror`, {
    detail: {message, error},
    bubbles: true, composed: true,
  }));
}
```
And then anywhere in tests you want to check for console erroring (or prevent WCT from failing on it) you can just stub that one method.